CMakeLists.txt

```diff
@@ -200,7 +200,7 @@ ELSE ()
     # Remove visibility flag for now as it breaks Ubuntu 18.05 and we need to
     # fix our libraries anyway
     STRING(REPLACE "-fvisibility=hidden" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
-    MY_CHECK_AND_SET_COMPILER_FLAG("-msse4.2" RELEASE RELWITHDEBINFO MINSIZEREL DEBUG)
+    MY_CHECK_AND_SET_COMPILER_FLAG("-msse4.2 -DBOOST_BIND_GLOBAL_PLACEHOLDERS" RELEASE RELWITHDEBINFO MINSIZEREL DEBUG)
```

Comment:
Why do we need this flag in this PR?

Reply:
I've removed this flag from the PR. However, it won't compile for me without it. THe new BOOST breaks itself.